examples/pinproctest/drivers.cpp

```diff
@@ -131,7 +131,7 @@ void ConfigureDriverGroups(PRHandle proc, bool driverPolarity)
     // in the matrixRowEnableIndex variables.
     const int mappedDriverGroupRowEnableSelect[] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1};
     
-    for (i = 0; i <= kPRDriverGroupsMax; i++)
+    for (i = 0; i < kPRDriverGroupsMax; i++)
```

Comment:
I caught this with a compiler warning.  The arrays referenced with index `i` only have `kPRDriverGroupsMax` entries, so we were reading past the ends of the arrays when creating `PRDriverGroupConfig` records.